godot-core/src/meta/rpc_config.rs

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use crate::builtin::{Dictionary, StringName};
+use crate::classes::multiplayer_api::RpcMode;
+use crate::classes::multiplayer_peer::TransferMode;
+use crate::classes::Node;
+use crate::dict;
+use crate::meta::ToGodot;
+
+/// See [Godot documentation](https://docs.godotengine.org/en/stable/tutorials/networking/high_level_multiplayer.html#remote-procedure-calls)
+#[derive(Clone, Copy, Debug)]
```

Comment:
Please apply [my suggestion](https://github.com/godot-rust/gdext/pull/902#discussion_r1766837389) verbatim (unless I make a mistake, of course). You can just click "Commit" on the web UI and squash later into your own commit.

The order should be `Copy, Clone, Debug`. I know it's bikeshedding, but it's [documented](https://godot-rust.github.io/book/contribute/conventions.html#types), and with my readily committable suggestion, the effort on your side to change should be minimal 😉 

(Yes, I know I _should_ automate this at some point...)